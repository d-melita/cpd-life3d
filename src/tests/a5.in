3 1024 .4 100
