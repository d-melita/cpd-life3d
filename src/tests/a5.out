1 99923786 1
2 90413714 1
3 83137654 1
4 77287897 1
5 72448825 1
6 68444736 1
7 65198270 1
8 62633412 1
9 60611199 1
